APMrover2/make.inc

```diff
@@ -38,6 +38,7 @@ LIBRARIES += AP_Navigation
 LIBRARIES += APM_Control
 LIBRARIES += AP_L1_Control
 LIBRARIES += AP_BoardConfig
+LIBRARIES += AP_InertialNav
```

Comment:
They are adding InertialNav to Rover and Plane so they need to add it to makefile...